packages/accounts/src/kernel-zerodev/__tests__/kernel-recovery-validator.test.ts

```diff
@@ -0,0 +1,167 @@
+  });
+
+  it("should sign hash properly", async () => {
+    expect(
+      await validator.signMessage("0xabcfC3DB1e0f5023F5a4f40c03D149f316E6A5cc")
+    ).eql(
+      "0x4d61c5c27fb64b207cbf3bcf60d78e725659cff5f93db9a1316162117dff72aa631761619d93d4d97dfb761ba00b61f9274c6a4a76e494df644d968dd84ddcdb1c"
+    );
+  });
+
+  it("should sign hash properly without 0x", async () => {
+    expect(await validator.signMessage("icanbreakthistestcase")).eql(
+      "0x4d61c5c27fb64b207cbf3bcf60d78e725659cff5f93db9a1316162117dff72aa631761619d93d4d97dfb761ba00b61f9274c6a4a76e494df644d968dd84ddcdb1c"
+    );
+  });
+
+  it(
+    "sendUserOperation should execute properly",
+    async () => {
+      let socialRecoveryProvider = await SocialRecoveryProvider.init({
+        projectId: config.projectId,
+        owner,
+        usePaymaster: false,
+        opts: {
+          providerConfig: {
+            opts: {
+              txMaxRetries: 10,
+            },
+          },
+        },
+      });
+
+      await socialRecoveryProvider.getAccount().getInitCode();
+
+      const result = socialRecoveryProvider.sendUserOperation({
+        target: "0x28a292f4dC182492F7E23CFda4354bff688f6ea8",
+        data: "0x",
+        value: 0n,
+      });
+      await expect(result).resolves.not.toThrowError();
+      await socialRecoveryProvider.waitForUserOperationTransaction(
+        (
+          await result
+        ).hash as Hash
+      );
+    },
+    { timeout: 100000 }
+  );
+
+  it(
+    "should correctly make an api call to set guardians",
+    async () => {
+      const data = {
+        guardians: {
+          "0x5B38Da6a701c568545dCfcB03FcB875f56beddC4": 100,
+        },
+        threshold: 50,
+        owneraddress: "0x7c8999dc9a822c1f0df42023113edb4fdd543266",
+      };
+
+      let socialRecoveryProvider = await SocialRecoveryProvider.init({
+        projectId: config.projectId,
+        owner,
+        usePaymaster: false,
+        opts: {
+          providerConfig: {
+            opts: {
+              txMaxRetries: 10,
+            },
+          },
+        },
+      });
+
+      const response = await validator.setGuardian(
+        data,
+        socialRecoveryProvider
+      );
+      expect(response).toBeDefined();
+      console.log("Set Guardian",response);
+      let recoveryid = response.recoveryid;
+      globalrecoveryid = recoveryid;
+    },
+    { timeout: 100000 }
+  );
+
+  it("should add signatures correctly", async () => {
+    const res = await validator.addSignatures(
+      globalrecoveryid,
+      "0xf0745420866c7ec0615a2fa25afaa271cd763596fb4b87fbde763f4cb9cfe142575c22419490fb9db86a6d18801c7919f49b9042619ee339ea200cd8ad533cf41b",
+      "0x7c8999dc9a822c1f0df42023113edb4fdd543266"
+    );
+    // console.log(res);
+    expect(res).toBeDefined();
+  });
+
+  it("should generate recovery calldata correctly", async () => {
+    try {
+      const res = await validator.getrecoveryCallData(
+        globalrecoveryid,
```

Comment:
Small thing but please use casing correctly, like `getRecoveryCalldata` and `globalRecoveryId`

Reply:
Especially for public APIs